splunk/src/main/java/com/splunk/OutputGroup.java

```diff
@@ -112,18 +112,18 @@ public void setCompressed(boolean compressed) {
     /**
      * Sets how long to wait before throwing out all new events until the output
      * queue has space. The default value is -1, which means to not drop events.
-     * <p><blockquote>
+     * <blockquote>
```

Comment:
- [ ] Let's capture this in the RELEASE_NOTES.md as "Fixes to Javadoc rendering"